test/usercommand/acl-decorator/MisconfiguredTsconfig.ts

```diff
@@ -0,0 +1,45 @@
+/* tslint:disable:no-console */
+import * as mage from 'mage'
+import * as assert from 'assert'
+import { Acl } from '../../../src'
+
+const sinon = require('sinon')
+
+let state: mage.core.IState
+let reflectStub: any
+
+describe('MisconfiguredTsconfig', function () {
+  before(() => {
+    state = new mage.core.State()
+    reflectStub  = sinon.stub(Reflect, 'getMetadata').returns(null)
+  })
+
+  after(() => {
+    reflectStub.restore()
+  })
+
+  // If experimentalDecorators or emitDecoratorMetadata are not set
+  // in tsconfig.json, Reflect.getMetadata returns null
+  //
+  // We test it correctly throws an error in this case
+  it('Throws an error if tsconfig is misconfigured', async function () {
+    try {
+    /**
+     * Empty class for testing purpose
+     *
+     * @class UserCommand
+     */
+      class UserCommand {
+        @Acl('*')
+        public static async execute(_state: mage.core.IState) {
+          return true
+        }
+      }
+      assert(UserCommand)
+    } catch (err) {
+      return
```

Comment:
Could you assert the content of the error?

Reply:
I could assert the error, but not its content as there is only a message: https://github.com/mage/mage-validator/blob/master/src/errors.ts#L32.

Reply:
You could assert that no?